src/components/cluster-form.tsx

```diff
@@ -408,7 +408,7 @@ export function ClusterForm() {
               name="worker_flavor"
               render={({ field }) => (
                 <FormItem>
-                  <FormLabel className="text-m">Worker Flavor</FormLabel>
+                  <FormLabel className="text-m">Worker Flavor <span className='border-2 rounded-3xl bg-gray-200 text-black px-2' title="SCS-( CPU )-( RAM[GiB] )-( DISK[GB] )">i</span></FormLabel>
```

Comment:
Good catch, we need to make a component :D

Reply:
Maybe [this one](https://ui.shadcn.com/docs/components/tooltip)